backend-diagram-converter/core/src/main/resources/converter-properties.properties

```diff
@@ -0,0 +1,44 @@
+# Zeebe Job Types
+## Adapter
+zeebe-job-type.adapter=camunda-7-adapter
+## Script
+zeebe-job-type.script=script
+# Zeebe Headers
+## Script
+zeebe-header.script=script
+## Result variable
+zeebe-header.result-variable=resultVariable
+## Adapter job type
+zeebe-header.adapter-job-type=camunda-7-adapter
+## Resource
+zeebe-header.resource=resource
+## Script format
+zeebe-header.script-format=scriptFormat
+# Namespaces
+## Zeebe
+namespace.zeebe.name=zeebe
+namespace.zeebe.uri=http://camunda.org/schema/zeebe/1.0
+## Conversion
+namespace.conversion.name=conversion
+namespace.conversion.uri=http://camunda.org/schema/conversion/1.0
+## Modeler
+namespace.modeler.name=modeler
+namespace.modeler.uri=http://camunda.org/schema/modeler/1.0
+## Camunda
+namespace.camunda.name=camunda
+namespace.camunda.uri=http://camunda.org/schema/1.0/bpmn
+## BPMN
+namespace.bpmn.name=bpmn
+namespace.bpmn.uri=http://www.omg.org/spec/BPMN/20100524/MODEL
+## XSI
+namespace.xsi.name=xsi
+namespace.xsi.uri=http://www.w3.org/2001/XMLSchema-instance
+# Static Attributes
```

Comment:
Are these namespaces relevant to configure them? 

Access in the code becomes a lot more effort:
```
context.getProperties().getCamundaNamespace().getUri()
```
before:
```
NamespaceUri.CAMUNDA
```

Reply:
Done